src/routes/settings/Plus.tsx

```diff
@@ -0,0 +1,198 @@
+import { Match, Show, Switch, createResource, createSignal } from "solid-js";
+import { A } from "solid-start";
+import { ConfirmDialog } from "~/components/Dialog";
+import { InfoBox } from "~/components/InfoBox";
+import NavBar from "~/components/NavBar";
+import {
+    Button,
+    DefaultMain,
+    LargeHeader,
+    MutinyWalletGuard,
+    NiceP,
+    SafeArea,
+    VStack
+} from "~/components/layout";
+import { BackLink } from "~/components/layout/BackLink";
+import { useMegaStore } from "~/state/megaStore";
+import eify from "~/utils/eify";
+
+import party from "~/assets/party.gif";
+
+function Perks(props: { alreadySubbed?: boolean }) {
+    return (
+        <ul class="list-disc ml-8 font-light text-lg">
+            <Show when={props.alreadySubbed}>
+                <li>Smug satisfaction</li>
+            </Show>
+            <li>
+                Redshift <em>(coming soon)</em>
+            </li>
+            <li>
+                Easy gifting <em>(coming soon)</em>
```

Comment:
```suggestion
                Gifting <em>(coming soon)</em>
```

I think just calling it gifting is better.